coding-challenges/README.md

```diff
@@ -111,3 +111,54 @@ Searching algorithm should correctly return the blogs that contain the search qu
 Test cases should cover basic scenarios and edge cases for the searching algorithm.
 
 ```
+
+3.Implement a Basic Linux Command Line Interface (CLI) Tool
+
+```bash
+**Context**
+
+To broaden the scope of learning and provide hands-on experience with command line tools, we want to introduce a challenge focused on building a basic Linux Command Line Interface (CLI) tool. This will allow beginners to get familiar with common Linux commands and to provide as a starting point to working with the terminal.
+
+**The Project**
+
+You are tasked with developing a simple CLI tool that performs basic file management tasks. Users should be able to interact with the tool via the command line to perform actions such as creating files, listing directory contents, renaming files, and deleting files.
+
+**Requirements**
+
+1. **CLI Commands**:
+    - Implement commands for basic file management tasks, including:
+    - `create`: Create a new file.
```

Comment:
Say we want to create a file with the path `libs/admin/index.ts` it should be able to create the folders in case they do not exist. It will make the challenge more interesting because this command does not exist in Linux.

Reply:
wdym? i thought there was something like `mkdir -p` which makes the folders if they dont exist, am i wrong?

Reply:
@AbuHafsa the command `mkdir -p /path/to/` allows you to create a directory along with any necessary parent directories that do not exist?

This is correct right? 

In my case this creates the folders if they dont exist

Reply:
You cannot create a file and the folders at the same time you have to do `mkdir -p /path/to/ && touch /path/to/index.ts`

Reply:
Ah I see OK, I'll add it in to the challenge shortly 👍